[DOCS] How to get currencies for channel (#14257)
Sylius/Sylius: docs/book/configuration/currencies.rst

        $currency = $this->get('sylius.context.currency')->getCurrency();
    }

Getting the list of available currencies for a channel
------------------------------------------------------

If you want to get a list of currently available currencies for a given channel,

> you can get them from the ``Channel``.
> You can also get the current ``Channel`` from the container.

Reviewer: ```suggestion
you can get it from the ``Channel``, get the current ``Channel`` from the container.
```

Reviewer 2: ```suggestion
you can get them from the ``Channel``, 
which can be retrieved from the container.
```
